plugins/inputs/zookeeper/zookeeper.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/influxdata/telegraf/plugins/inputs"
 )
 
-var zookeeperFormatRE = regexp.MustCompile(`^zk_(\w+)\s+([\w\.\-]+)`)
+var zookeeperFormatRE = regexp.MustCompile(`^zk_([\w\.\-]+)\s+([\w\.\-]+)`)
```

Comment:
I think this is not correct as you now also accept items with "." or "-" only. I guess the correct regex should be
```suggestion
var zookeeperFormatRE = regexp.MustCompile(`^zk_(\w[\w\.\-]*)\s+([\w\.\-]+)`)
```